src/ScheduleFunctions.cpp

```diff
@@ -2243,7 +2287,14 @@ bool validate_schedule(Function f, const Stmt &s, const Target &target, bool is_
         err << "Func \"" << f.name() << "\" is computed at the following invalid location:\n"
             << "  " << schedule_to_source(f, store_at, compute_at) << "\n"
             << "Legal locations for this function are:\n";
-        for (const auto &site : sites) {
+        for (size_t i = 0; i < sites.size(); i++) {
+            const auto &site = sites[i];
+            if (invalid_sites.count(i)) {
+                if (debug::debug_level() > 0) {
```

Comment:
Was this intentionally left in?

Reply:
Yes, though it probably should be pruned (see comment below)